packages/client/package.json

```diff
@@ -13,15 +13,16 @@
   "npmClient": "npm",
   "scripts": {
     "check-errors": "tsc --noemit",
-    "dev": "cross-env APP_ENV=development NODE_OPTIONS=--max_old_space_size=20480 vite",
-    "build": "cross-env NODE_OPTIONS=--max_old_space_size=10240 vite build",
+    "dev": "npm run update-root-cookie-accessor-domain && cross-env APP_ENV=development NODE_OPTIONS=--max_old_space_size=20480 vite",
```

Comment:
we should include this script inside the vite.config.ts instead of polluting the commands here